modules/appeals_api/app/swagger/appeals_api/v2/api_description.md

```diff
@@ -70,10 +70,9 @@ Example `metadata.json` file:
 }
 ```
 The Decision Review API will set the businessLine for your Evidence submission to ensure the documents are routed to the correct group within VA.
-You may check the status of your evidence document upload by using `GET` `/notice_of_disagreements/evidence_submissions/{uuid}` or `GET` `/supplemental_claims/evidence_submissions/{uuid}`. If, after you've uploaded a document, the status hasn't changed to `uploaded` before 15 minutes has elapsed, we recommend retrying the submission to make sure the document properly reaches our servers.
+You may check the status of your evidence document upload by using `GET` `/notice_of_disagreements/evidence_submissions/{uuid}`. If, after you've uploaded a document, the status hasn't changed to `uploaded` before 15 minutes has elapsed, we recommend retrying the submission to make sure the document properly reaches our servers.
 
 For NODs, evidence may only be uploaded within 90 days of the NOD reaching submitted status. After 90 days an error will be returned if evidence uploads related to this NOD are attempted.
-For Supplemental Claims, evidence must be uploaded immediately after the SC reaching submitted status. After 2 days an error will be returned if evidence uploads related to this SC are attempted.
```

Comment:
I removed references to SC for this file to serve as an example; petending we hadn't already pushed SC docs to prod.